pkg/apis/acid.zalan.do/v1/postgresql_type.go

```diff
@@ -53,8 +53,8 @@ type PostgresSpec struct {
 	Databases             map[string]string    `json:"databases,omitempty"`
 	Tolerations           []v1.Toleration      `json:"tolerations,omitempty"`
 	Sidecars              []Sidecar            `json:"sidecars,omitempty"`
-	InitContainers        []v1.Container       `json:"init_containers,omitempty"`
-	PodPriorityClassName  string               `json:"pod_priority_class_name,omitempty"`
+	InitContainers        []v1.Container       `json:"initContainers,omitempty"`
+	PodPriorityClassName  string               `json:"podPriorityClassName,omitempty"`
```

Comment:
Tested with Postgres cluster using the old style and updated the image in the deployment. After rolling update both were still in place. However, when creating new clusters with snake_case styled parameters it is not taken into account. So, I could use [multiple JSON tags](https://stackoverflow.com/questions/18635671/how-to-define-multiple-name-tags-in-a-struct/18635910) for backwards compatibility.

Reply:
Simply having multiple `json` tags doesn't work. Only via another struct. So, lets keep it as it is for now and change on v2.0 release.